### ISSUE TITLE
ClickHouse Table Migration Broken By PR#8255 For ClickHouse Versions < 2.7

### ISSUE THREAD

**Issue by ethankelly99** [cloudquery/cloudquery#8400]
### Describe the Bug

The table migration fails in version v1.1.0 of the ClickHouse destination plugin for versions of ClickHouse earlier than 22.7. It fails due to a change in #8255 which changed the _tables.go_ `CreateTable` method and introduced a trailing comma to the `CREATE TABLE` statement. Support for trailing commas in CREATE TABLE statements was added to ClickHouse 22.7 in ClickHouse/ClickHouse#38440. 

The following warning line is printed in the CloudQuery logs
```
WARNING: version 22.6.9 of ClickHouse is not supported by this client
```
But the CloudQuery documentation for the ClickHouse destination says versions >= 22 are supported.

@candiduslynx already commented on the behaviour in [a review comment](https://github.com/cloudquery/cloudquery/pull/8255#discussion_r1111920337) on the PR

### Expected Behavior

Tables should be created successfully in all supported ClickHouse versions.

### CloudQuery Version

2.3.10
### Debug Output

[cloudquery.log](https://github.com/cloudquery/cloudquery/files/10812087/cloudquery.log)

### Steps to Reproduce

Run `cloudquery sync` with the ClickHouse destination version v1.1.0 against a version of ClickHouse earlier than 2.7 (e.g. 2.6).

Here's the destination plugin configuration
```yaml
kind: destination
spec:
  name: "clickhouse"
  path: cloudquery/clickhouse
  version: "v1.1.0"
  write_mode: "append"
  spec:
    connection_string: "clickhouse://default:@localhost:9999/cloudquery?debug=true"
```

### Additional Context

I modified the CreateTable function locally to remove the trailing comma and this resolved the issue.

### References

#8255

**Comment by erezrokah** [comment]
Thanks for the issue @ethankelly99. I'll create a fix in a bit

**Comment by erezrokah** [comment]
Hi @ethankelly99, sorry for the trouble. We released [a new v1.1.1 version](https://github.com/cloudquery/cloudquery/releases/tag/plugins-destination-clickhouse-v1.1.1) with a fix to this issue, and also updated our CI to test with the minimal 22 version (we were testing on a newer version).

**Comment by ethankelly99** [comment]
Hi @erezrokah, no worries! Thanks for the awesome turn-around time 💯
